js/app.js

```diff
@@ -0,0 +1,69 @@
+// Section 1 
+
+// interpolation is combining strings together with a variable in the console for example const age = 3 you could write "Im " + age + " years old!"
+// concatentation is combining two strings together "Chicago" + "City". This would compute out to Chicago City when you put it into console.log
+
+// D.R.Y stands for Don't repeat yourself. anytime in coding when you have repeated yourself two or more times it is wrong. It is a waste of space and an complicate code for no reason
+
+// Section 2
+
+ const a = 4;
+ const b = 53;
+ const c = 57;
+ const d = 16;
+ const e = 'Kevin';
+
+console.log(a < b);
+console.log(c > d);
+console.log('Name' + 'Name');
+console log(a + b = c);
```

Comment:
did you try this in your console?  `=` is an assignment operator.  you can't assign a value to `a+b`

you should be trying this stuff in your own console before submitting!  make it log "true"